### PR TITLE
fix : updated inputs

### PR DESCRIPTION
**Pull request by prankurpandeyy**

### CODE REVIEW

**Comment by Naman-Saxena1** [comment]
Looks fine.

**Comment by rahulk31** [comment]
Looks good, can be merged.

**Review by varun1233 — APPROVED** [review]
Code looks good

File: components/Input/input.css [hunk 1]
```diff
@@ -60,13 +60,13 @@ input + span {
   
   
   input:invalid + span::before {
-    content: '✖';
+    /* content: '✖'; */
```
Comment (sri-rishi) [inline components/Input/input.css:71]: remove comments

**Review by sri-rishi — CHANGES_REQUESTED** [review]

**Pull request merged**